examples/celery-rabbitmq/rabbitmq-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  labels:
+    name: rabbitmq-service
+  name: rabbitmq-service
+spec:
+  port: 5672
```

Comment:
Without the port being there, I get this error: "Client error: Service "rabbitmq-service" is invalid: spec.port: invalid value '0': must be greater than 0 and less than 65536"

When it's there, everything works fine.